core/src/test/java/jenkins/model/lazy/FakeMapBuilder.java

```diff
@@ -48,7 +49,7 @@ public FakeMapBuilder() {
 
     public FakeMapBuilder add(int n) throws IOException {
         File build = new File(dir, Integer.toString(n));
-        FileUtils.writeStringToFile(new File(build, "n"), Integer.toString(n));
+        FileUtils.writeStringToFile(new File(build, "n"), Integer.toString(n), StandardCharsets.US_ASCII);
```

Comment:
Again why not UTF_8?

Reply:
Because the content is in fact guaranteed to be ASCII.